commercial/app/views/hosted/guardianHostedGallery.scala.html

```diff
@@ -7,32 +7,36 @@
         @guardianHostedHeader("hosted-gallery-page", page.logoUrl)
         <div class="l-side-margins hosted__side">
             <section class="hosted-tone--dark hosted-gallery__gallery-section">
-                <div class="hosted-gallery__image hosted-tone--dark" style="opacity: 1; background-image: url(@{page.images.head.url});">
-                    <div class="hosted-gallery__meta">
-                        <h1 class="hosted-gallery__heading">@{page.title}</h1>
-                        <h3 class="hosted-gallery__heading">@{page.standfirst}</h3>
-                    </div>
-                </div>
-                @for(i <- page.images.indices.tail) {
-                    <div class="hosted-gallery__image hosted-tone--dark" style="opacity: 0; background-image: url(@{page.images(i).url});"></div>
-                }
-                <div class="hosted-gallery__scroll-container">
-                    @for(i <- page.images.indices) {
-                        <div class="hosted-gallery__image--placeholder"></div>
-                    }
-                </div>
-                <div class="hosted-gallery__progress">
-                    @fragments.inlineSvg("arrow-up", "icon")
-                    <div class="hosted-gallery__progress--wrapper">
-                        <div class="hosted-gallery__progress--border">
-                            <div class="hosted-gallery__progress--border-1"></div>
-                            <div class="hosted-gallery__progress--border-2"></div>
+                <div class="js-hosted-gallery-container">
+                    <div class="js-hosted-gallery-images">
+                        <div class="hosted-gallery__image hosted-tone--dark" style="background-image: url(@{page.images.head.url});">
+                            <div class="hosted-gallery__meta">
+                                <h1 class="hosted-gallery__heading">@{page.title}</h1>
+                                <h3 class="hosted-gallery__heading">@{page.standfirst}</h3>
+                            </div>
                         </div>
+                        @for(i <- page.images.indices.tail) {
+                            <div class="hosted-gallery__image hosted-tone--dark"></div>
+                        }
+                    </div>
+                    <div class="hosted-gallery__scroll-container">
+                        @for(i <- page.images.indices) {
+                            <div class="hosted-gallery__image--placeholder"></div>
+                        }
                     </div>
-                    <div class="hosted-gallery__image-count">
-                        1/@page.images.length
+                    <div class="hosted-gallery__progress">
+                        @fragments.inlineSvg("arrow-up", "icon")
+                        <div class="hosted-gallery__progress--wrapper">
```

Comment:
I would probably name it `hosted-gallery__progress-wrapper`, because (afaik) `wrapper` is not a modifier. But maybe ask someone who understands BEM more than I do ;)